test/remote/gateways/remote_priority_test.rb

```diff
@@ -24,37 +24,37 @@ def setup
     @option_spr = {
       billing_address: address(),
       invoice: '666',
-      cardPresent: false,
-      cardPresentType: 'CardNotPresent',
-      isAuth: false,
-      paymentType: 'Sale',
-      bankAccount: '',
-      shouldVaultCard: false,
-      taxExempt: false,
-      tenderType: 'Card',
+      card_present: 'false',
```

Comment:
It looks like these same fields are used in the unit test file, so they should be updated there also please.

Reply:
@meagabeth can you please specify what needs to be done here?

Reply:
The @option_spr variable in the unit test file should be updated to match the format of the fields in the @options_spr variable here in the remote test

Reply:
@meagabeth changed